### PR TITLE
[Localization] Localization batch - [Not meant to be merged until last minute]

### PR DESCRIPTION
**Pull request by Adri1**
## What are the changes the user will see?
New translated / adjusted / corrected text

## Why am I making these changes?
To have these texts translated / adjusted / corrected

## What are the changes from a developer perspective?
None, just translated / adjusted / corrected text

## Checklist
- [X] **I'm using `beta` as my base branch**
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?

### CODE REVIEW

File: src/locales/ko/battler-tags.json [hunk 1]
```diff
@@ -69,5 +69,6 @@
   "cursedLapse": "{{pokemonNameWithAffix}}[[는]]\n저주받고 있다!",
   "stockpilingOnAdd": "{{pokemonNameWithAffix}}[[는]]\n{{stockpiledCount}}개 비축했다!",
   "disabledOnAdd": "{{pokemonNameWithAffix}}의 {{moveName}}[[는]]\n사용할 수 없다!",
-  "disabledLapse": "{{pokemonNameWithAffix}}의 {{moveName}}[[는]]\n이제 사용할 수 있다."
+  "disabledLapse": "{{pokemonNameWithAffix}}의 {{moveName}}[[는]]\n이제 사용할 수 있다.",
+  "tarShotOnAdd": "{{pokemonNameWithAffix}} became weaker to fire!"
```
Comment (sodaMelon) [inline src/locales/ko/battler-tags.json:73]:
```suggestion
  "tarShotOnAdd": "{{pokemonNameWithAffix}}[[는]] 불꽃에 약해졌다!"
```
image

Comment (EnochG1) [inline src/locales/ko/battler-tags.json:73]:
```suggestion
  "tarShotOnAdd": "{{pokemonNameWithAffix}}[[는]]불꽃에 약해졌다!"
```

File: src/locales/pt_BR/battler-tags.json [hunk 2]
```diff
@@ -67,5 +67,6 @@
   "saltCuredLapse": "{{pokemonNameWithAffix}} foi ferido pelo {{moveName}}!",
   "cursedOnAdd": "{{pokemonNameWithAffix}} cortou seus PS pela metade e amaldiçoou {{pokemonName}}!",
   "cursedLapse": "{{pokemonNameWithAffix}} foi ferido pelo Curse!",
-  "stockpilingOnAdd": "{{pokemonNameWithAffix}} estocou {{stockpiledCount}}!"
+  "stockpilingOnAdd": "{{pokemonNameWithAffix}} estocou {{stockpiledCount}}!",
+  "tarShotOnAdd": "{{pokemonNameWithAffix}} became weaker to fire!"
```
Comment (josericardo-fo) [inline src/locales/pt_BR/battler-tags.json:71]:
```suggestion
  "tarShotOnAdd": "{{pokemonNameWithAffix}} tornou-se mais fraco ao fogo!"
```

File: src/locales/ja/battler-tags.json [hunk 3]
```diff
@@ -69,5 +69,6 @@
   "cursedLapse": "{{pokemonNameWithAffix}}は　のろわれている！",
   "stockpilingOnAdd": "{{pokemonNameWithAffix}}は　{{stockpiledCount}}つ　たくわえた!",
   "disabledOnAdd": "{{pokemonNameWithAffix}}の\n{{moveName}}\nを　封じこめた！",
-  "disabledLapse": "{{pokemonNameWithAffix}}の\nかなしばりが　解けた！"
+  "disabledLapse": "{{pokemonNameWithAffix}}の\nかなしばりが　解けた！",
+  "tarShotOnAdd": "{{pokemonNameWithAffix}} became weaker to fire!"
```
Comment (Chapybara-jp) [inline src/locales/ja/battler-tags.json:73]:
```suggestion
  "tarShotOnAdd": "{{pokemonNameWithAffix}}は　ほのおに　弱くなった！"
```

File: src/locales/es/battler-tags.json [hunk 4]
```diff
@@ -69,5 +69,6 @@
   "cursedLapse": "¡{{pokemonNameWithAffix}} es víctima de una maldición!",
   "stockpilingOnAdd": "¡{{pokemonNameWithAffix}} ha reservado energía por {{stockpiledCount}}ª vez!",
   "disabledOnAdd": "¡Se ha anulado el movimiento {{moveName}}\nde {{pokemonNameWithAffix}}!",
-  "disabledLapse": "¡El movimiento {{moveName}} de {{pokemonNameWithAffix}} \n ya no está anulado!"
+  "disabledLapse": "¡El movimiento {{moveName}}\nde {{pokemonNameWithAffix}} ya no está anulado!",
```
Comment (AsdarDevelops) [inline src/locales/es/battler-tags.json:72]:
```suggestion
  "disabledLapse": "¡El movimiento {{moveName}} de {{pokemonNameWithAffix}} ya no está anulado!",
```

I don't want to risk overflowing to 3rd line here

**Review by AsdarDevelops — CHANGES_REQUESTED** [review]

File: src/locales/it/battler-tags.json [hunk 5]
```diff
@@ -69,5 +69,6 @@
   "cursedLapse": "{{pokemonNameWithAffix}} subisce la maledizione!",
   "stockpilingOnAdd": "{{pokemonNameWithAffix}} ha usato Accumulo per la\n{{stockpiledCount}}ª volta!",
   "disabledOnAdd": "La mossa {{moveName}} di\n{{pokemonNameWithAffix}} è stata bloccata!",
-  "disabledLapse": "La mossa {{moveName}} di\n{{pokemonNameWithAffix}} non è più bloccata!"
+  "disabledLapse": "La mossa {{moveName}} di\n{{pokemonNameWithAffix}} non è più bloccata!",
+  "tarShotOnAdd": "{{pokemonNameWithAffix}} became weaker to fire!"
```
Comment (NicusPulcis) [inline src/locales/it/battler-tags.json:73]:
```suggestion
  "tarShotOnAdd": "{{pokemonNameWithAffix}} è diventato vulnerabile\nal tipo Fuoco!"
```

**Review by NicusPulcis — APPROVED** [review]
italian localization done (added a \n to make sure it doesnt exceed the border limits)

File: src/locales/de/battler-tags.json [hunk 6]
```diff
@@ -69,5 +69,6 @@
   "cursedLapse": "{{pokemonNameWithAffix}} wurde durch den Fluch verletzt!",
   "stockpilingOnAdd": "{{pokemonNameWithAffix}} hortet {{stockpiledCount}}!",
   "disabledOnAdd": " {{moveName}} von {{pokemonNameWithAffix}} wurde blockiert!",
-  "disabledLapse": "{{moveName}} von {{pokemonNameWithAffix}} ist nicht länger blockiert!"
+  "disabledLapse": "{{moveName}} von {{pokemonNameWithAffix}} ist nicht länger blockiert!",
+  "tarShotOnAdd": "{{pokemonNameWithAffix}} became weaker to fire!"
```
Comment (CodeTappert) [inline src/locales/de/battler-tags.json:73]:
```suggestion
  "tarShotOnAdd": "{{pokemonNameWithAffix}} ist nun schwach gegenüber Feuer-Attacken!"
```

File: src/locales/pt_BR/fight-ui-handler.json [hunk 7]
```diff
@@ -3,5 +3,6 @@
   "power": "Poder",
   "accuracy": "Precisão",
   "abilityFlyInText": " {{passive}}{{pokemonName}}\n{{abilityName}}",
-  "passive": "Passiva de "
-}
\ No newline at end of file
+  "passive": "Passiva de ",
+  "teraHover": "{{type}} Terastallized"
```
Comment (josericardo-fo) [inline src/locales/pt_BR/fight-ui-handler.json:7]:
```suggestion
  "teraHover": "Terastalizado {{type}}"
```

File: src/locales/es/move-trigger.json [hunk 8]
```diff
@@ -12,5 +12,6 @@
   "stoleItem": "¡{{pokemonName}} robó el objeto\n{{itemName}} de {{targetName}}!",
   "statEliminated": "¡Los cambios en estadísticas fueron eliminados!",
   "revivalBlessing": "¡{{pokemonName}} ha revivido!",
-  "safeguard": "¡{{targetName}} está protegido por Velo Sagrado!"
+  "safeguard": "¡{{targetName}} está protegido por Velo Sagrado!",
+  "afterYou": "{{pokemonName}} took the kind offer!"
```
Comment (DanStevensonx) [inline src/locales/es/move-trigger.json:16]:
```suggestion
  "afterYou": "¡{{pokemonName}} ha decidido aprovechar la oportunidad!"
```

File: src/locales/es/fight-ui-handler.json [hunk 9]
```diff
@@ -3,5 +3,6 @@
   "power": "Potencia",
   "accuracy": "Precisión",
   "abilityFlyInText": " {{passive}}{{pokemonName}}\n{{abilityName}}",
-  "passive": "Pasiva de "
+  "passive": "Pasiva de ",
+  "teraHover": "{{type}} Terastallized"
```
Comment (DanStevensonx) [inline src/locales/es/fight-ui-handler.json:7]:
```suggestion
  "teraHover": "{{type}} se teracristalizó"
```

**Review by DanStevensonx — COMMENTED** [review]
Suggested the Spanish translation. Line 7 of fight-ui-handler.json didn't have a translation Corpus, so I translated at my own discretion.

File: src/locales/ko/fight-ui-handler.json [hunk 10]
```diff
@@ -3,5 +3,6 @@
   "power": "위력",
   "accuracy": "명중률",
   "abilityFlyInText": " {{pokemonName}}의 {{passive}}{{abilityName}}",
-  "passive": "패시브 "
-}
\ No newline at end of file
+  "passive": "패시브 ",
+  "teraHover": "{{type}} Terastallized"
```
Comment (EnochG1) [inline src/locales/ko/fight-ui-handler.json:7]:
```suggestion
  "teraHover": "{{type}} 테라스탈"
```

File: src/locales/ko/move-trigger.json [hunk 11]
```diff
@@ -66,5 +66,6 @@
   "revivalBlessing": "{{pokemonName}}[[는]]\n정신을 차려 싸울 수 있게 되었다!",
   "swapArenaTags": "{{pokemonName}}[[는]]\n서로의 필드 효과를 교체했다!",
   "exposedMove": "{{pokemonName}}[[는]]\n{{targetPokemonName}}의 정체를 꿰뚫어 보았다!",
-  "safeguard": "{{targetName}}[[는]] 신비의 베일이 지켜 주고 있다!"
+  "safeguard": "{{targetName}}[[는]] 신비의 베일이 지켜 주고 있다!",
+  "afterYou": "{{pokemonName}} took the kind offer!"
```
Comment (EnochG1) [inline src/locales/ko/move-trigger.json:70]:
```suggestion
  "afterYou": "{{pokemonName}}[[는]]\n배려를 받아들이기로 했다!"
```

**Review by NicusPulcis — DISMISSED** [review]
fixing the terastilized text in italian

File: src/locales/pt_BR/move-trigger.json [hunk 12]
```diff
@@ -62,5 +62,6 @@
   "revivalBlessing": "{{pokemonName}} foi reanimado!",
   "swapArenaTags": "{{pokemonName}} trocou os efeitos de batalha que afetam cada lado do campo!",
   "exposedMove": "{{pokemonName}} identificou\n{{targetPokemonName}}!",
-  "safeguard": "{{targetName}} está protegido por Safeguard!"
+  "safeguard": "{{targetName}} está protegido por Safeguard!",
+  "afterYou": "{{pokemonName}} took the kind offer!"
```
Comment (josericardo-fo) [inline src/locales/pt_BR/move-trigger.json:66]:
```suggestion
  "afterYou": "{{pokemonName}} aceitou a gentil oferta!"
```

File: src/locales/ja/fight-ui-handler.json [hunk 13]
```diff
@@ -2,6 +2,7 @@
   "pp": "PP",
   "power": "威力",
   "accuracy": "命中",
-  "abilityFlyInText": " {{pokemonName}}の\n{{passive}} {{abilityName}}",
-  "passive": "パッシブ "
+  "abilityFlyInText": " {{pokemonName}}の\n{{passive}}{{abilityName}}",
+  "passive": "パッシブ ",
+  "teraHover": "{{type}} Terastallized"
```
Comment (EnochG1) [inline src/locales/ja/fight-ui-handler.json:7]:
```suggestion
  "teraHover": "{{type}}テラスタル"
```

**Review by Chapybara-jp — DISMISSED** [review]

File: src/locales/zh_CN/fight-ui-handler.json [hunk 14]
```diff
@@ -2,6 +2,7 @@
   "pp": "PP",
   "power": "威力",
   "accuracy": "命中",
-  "abilityFlyInText": " {{pokemonName}} 的 {{passive}}{{abilityName}}",
-  "passive": "被动 "
-}
\ No newline at end of file
+  "abilityFlyInText": " {{pokemonName}} 的\n{{passive}}{{abilityName}}",
+  "passive": "被动 ",
+  "teraHover": "{{type}} Terastallized"
```
Comment (40chyan) [inline src/locales/zh_CN/fight-ui-handler.json:7]:
```suggestion
  "teraHover": "{{type}} 太晶化"
```

File: src/locales/zh_TW/fight-ui-handler.json [hunk 15]
```diff
@@ -2,6 +2,7 @@
   "pp": "PP",
   "power": "威力",
   "accuracy": "命中率",
-  "abilityFlyInText": " {{pokemonName}} 的 {{passive}}{{abilityName}}",
-  "passive": "被動能力 "
-}
\ No newline at end of file
+  "abilityFlyInText": " {{pokemonName}} 的\n{{passive}}{{abilityName}}",
+  "passive": "被動能力 ",
+  "teraHover": "{{type}} Terastallized"
```
Comment (40chyan) [inline src/locales/zh_TW/fight-ui-handler.json:7]:
```suggestion
  "teraHover": "{{type}} 太晶化"
```

**Review by 40chyan — COMMENTED** [review]
zn tera

**Review by SangaraSorama — APPROVED** [review]

**Review by CodeTappert — APPROVED** [review]

**Review by DayKev — APPROVED** [review]

**Pull request merged**